src/Appwrite/Platform/Workers/Messaging.php

```diff
@@ -17,6 +18,7 @@
 use Utopia\Platform\Action;
 use Utopia\Queue\Message;
 use Appwrite\Event\Usage;
+use libphonenumber\PhoneNumberUtil;
```

Comment:
Let's not use the lib directly in Appwrite and instead abstract it in utopia/messaging.